packages/react-core/src/hooks/__tests__/useIsSignedIn.test.ts

```diff
@@ -0,0 +1,90 @@
+import useIsSignedIn from '../useIsSignedIn';
+import { act, renderHook } from '@testing-library/react-hooks';
+import { getCurrentUser } from 'aws-amplify/auth';
+import { GetCurrentUserOutput } from 'aws-amplify/auth';
+import { Hub } from 'aws-amplify/utils';
+
+jest.mock('aws-amplify/auth', () => ({
+  getCurrentUser: jest.fn(),
+}));
+
+describe('useIsSignedIn', () => {
+  it('should be falsy if user is not signed in', () => {
```

Comment:
Noted below as well, but do not use "falsy"/"truthy" when checking for actual boolean values in assertions and tst names